### ISSUE TITLE
Missing 'toSQL' for SchemaBuilder

### ISSUE THREAD

**Issue by tflanagan** [knex/knex#3751]
# Environment

Knex version: 0.20.12
Database + version: pg@7.18.2
OS: Windows 10

If issue is about TypeScript definitions, tag @lorefnon 

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do

Attempting to call `db.schema.raw('sql statement').toSQL()`.

Expecting it to be there (works without using ts). http://knexjs.org/#Interfaces-toSQL

2. Error message

Receiving type error:

`Property 'toSQL' does not exist on type 'SchemaBuilder'.`

3. Reduced test code, for example in https://npm.runkit.com/knex or if it needs real
   database connection to MySQL or PostgreSQL, then single file example which initializes
   needed data and demonstrates the problem.

```ts
import knex from 'knex';

const db = knex({ ... });

db.schema.raw('sql statement').toSQL();
```

**Comment by tflanagan** [comment]
Thanks for the quick turn around!

**Comment by kibertoad** [comment]
@tflanagan I'll cherry-pick latest fixes tomorrow and release them in 0.20.x branch

**Comment by kibertoad** [comment]
Released in  0.20.14.

**Comment by uiii** [comment]
Actually the return type of `toSQL` is not `Sql` object but and array of objects containing non-native sql but without `toNative` method. Could you please correct this?
